crates/bevy_winit/src/system.rs

```diff
@@ -137,6 +148,8 @@ pub struct CachedWindow {
 /// - [`Window::focused`] cannot be manually changed to `false` after the window is created.
 pub(crate) fn changed_windows(
```

Comment:
(I apologise if I misunderstand but)
Is `changed_windows` the correct system for the task?
At least from the docstring I get the idea one should take data from `Window` and update the corresponding winit window to match the users settings rather than reading data from winit to expose them to the user.
Especially given the Query has a `Changed<Window>` therefore the loop checking for the events would run only when the user changes the window parameters. Is that the desired behaviour?

Reply:
As far as I've understood from this code, we're doing the inverse here. We take minimize/maximize requests (hence whi the Changed filter should work) and set the winit window. To me it made sense to trigger these events from here too. But your callout is very valid. I don't feel confident about refactoring all of this behavior though and feels like we could revisit it at some point.

Reply:
I was just able to test this out properly now and the events aren't working as intended so my assumptions were wrong. I'll dig into this a bit more and come up with a working solution.